CCL.Importer/PrefabWrangler.cs

```diff
@@ -65,10 +69,66 @@ private static bool FinalizeLiveryPrefab(CCL_CarVariant livery)
             newFab.name = livery.id;
             livery.prefab = newFab;
 
+
+            BuildSimulationElements(livery);
+
             CCLPlugin.Log($"Finalized prefab for {livery.id}");
             return true;
         }
 
+        //==============================================================================================================
+        #region SimulationComponents - SimController and friends
+        private static void BuildSimulationElements(TrainCarLivery livery)
+        {
+            AddAdditionalControllers(livery.prefab);
+            if (livery.interiorPrefab != null)
```

Comment:
For anything inheriting from UnityEngine.Object we can use the implicit bool conversion - `if (livery.interiorPrefab)`, I think it's recommended over a check against null

Reply:
I was unaware of that capability - I'll make those changes later today